website/cue/reference/components/sinks/blackhole.cue

```diff
@@ -13,7 +13,7 @@ components: sinks: blackhole: {
 	}
 
 	features: {
-		acknowledgements: true
+		acknowledgements: false
```

Comment:
Was this suppose to be the `auto_generated` option? Or did I miss something here?

Reply:
This flag drops the acknowledgements docs from automatically being added.

I thought hard about this flag. I went with having the code be the source of truth since that's the purpose of this entire project.

Reply:
🤔 I thought that was one of the things that got disabled when `auto_generated: true`

Reply:
ah let me look into that...

Reply:
yep, auto_generated does the same thing. thanks!